---
section: value-stories
lang: en
title:  "Hong Kong/China - open sourcing genomes / crowdsourcing killer outbreaks"
authors:
- S.C. Edmunds
---
The genome from the deadly 2011 E. coli in Germany was the first dataset we at GigaScience released with a [DOI](http://en.wikipedia.org/wiki/Digital_object_identifier) and [CC0](http://en.wikipedia.org/wiki/CC0#Public_domain_tools) waiver.  Due to the unusual severity of the outbreak â€“ thousands severely ill and over 50 deaths, it was clear that the usual scientific procedure of producing data, analyzing it slowly and then releasing it to the public after a potentially long peer-review procedure would not have been helpful in this case. By releasing the first genomic data before it had even finished uploading to the usual scientific repositories ([NCBI](http://www.ncbi.nlm.nih.gov/sites/entrez?cmd=Link&LinkName=bioproject_sra&from_uid=67657)) via [twitter](http://sciblogs.co.nz/infectious-thoughts/2011/06/09/an-outbreak-of-crowdsourcing/), promoting its use, and releasing [subsequently](http://www.genomics.cn/en/news/show_news?nid=98856) [improved](http://www.genomics.cn/en/news/show_news?nid=98867) assemblies in this way, a huge community of microbial genomicists from around the world took up the challenge to study the organism collaboratively (the process was  [dubbed](http://twitter.com/#%21/pathogenomenick/status/78768010382422016) by some as the first "Tweenome"). A [github](https://github.com/ehec-outbreak-crowdsourced/BGI-data-analysis/wiki) repository was created (thanks to the efforts of the [Era7](http://www.era7bioinformatics.com/en/index.html) team in Spain) to provide a home for the analyses and data and within 24 hours groups from around the world started producing their own annotations and assemblies. Within a few days a potential ancestral strain was [identified by another blogger](http://scienceblogs.com/mikethemadbiologist/2011/06/i_dont_think_the_german_e_coli.php), helping to clear Spanish farmers of the blame and end the massive boycotting of their agricultural products. 

The many eyes thrown at this data allowed the antibiotic resistance genes and pathogenic features to be much more clearly understood. Obviously the main aim of doing science in this accelerated way was speed up diagnosis and treatments during a serious health crisis, and the E. coli data enabled the rapid development of [diagnostic tests](http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0034498) and [anti-microbial agents](http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0033637). Within less than a week of the first data coming out a [free diagnostic protocol and free primers](http://www.genomics.cn/en/news/show_news?nid=98968) were distributed by the BGI to immediately help track the source of the outbreak. This rapid and open response seems an obvious thing to do in a health crisis such as this, but is shockingly still not the norm when contrasted with the [lack of data sharing from the Ebola crisis](http://www.nature.com/news/data-sharing-make-outbreak-research-open-access-1.16966).

Releasing the data under a [CC0](http://en.wikipedia.org/wiki/CC0#Public_domain_tools) license allowed truly open-source analysis. The [UK HPA](http://www.hpa-bioinformatics.org.uk/lgp/genomes) and other researchers [followed suit](http://bacpathgenomics.wordpress.com/2011/06/13/e-coli-data-released-under-creative-commons-0-license/) by releasing their work in this way. Following this example, a team at Pacific-Biosystems also released their related data in a similar manner, using [the example](http://www.nature.com/news/open-data-project-aims-to-ease-the-way-for-genomic-research-1.10507) of their fellow E. coli data producers, which allowed them to release their data without wasting time on legal wrangling. This has subsequently been used as an example for future UK and EU science policy, with the Royal Society using the E. coli crowdsourcing as an example of "the power of intelligently open data" and highlighting it on the cover of their influential ["Science as an Open Enterprise"](http://royalsociety.org/policy/projects/science-public-enterprise/report/) report. 

For more information about this open science crowdsourcing effort, you can read the following blog posts: 
* [Notes from an E. coli "tweenome" - Lessons Learned from our First Data DOI](http://blogs.biomedcentral.com/gigablog/2011/08/03/notes-from-an-e-coli-tweenome-lessons-learned-from-our-first-data-doi/)
* [Open Science versus Ash Dieback and the Tweenome Revisited](http://blogs.biomedcentral.com/gigablog/2013/02/13/open-science-versus-ash-dieback-and-the-tweenome-revisited/) 
* [Tweenome on Film: Excellent Video on Crowdsourcing Killer Outbreaks](http://blogs.biomedcentral.com/gigablog/2013/02/21/tweenome-on-film-excellent-video-on-crowdsourcing-killer-outbreaks/) 
* [BBSRC Video Discussing the Crowdsourcing Effort](http://www.bbsrc.ac.uk/news/food-security/2013/130218-f-crowdsourcing-killer-outbreaks/)
